scripts/provisioning.sh

```diff
@@ -24,7 +24,7 @@ if [[ ! -f .debops.cfg && -f ../.debops.cfg ]]; then
 fi
 
 # Run the bootstrap DebOps playbook to make the VM a DebOps/Ansible managed host.
-debops $@ bootstrap-ldap || debops --user root $@ bootstrap-ldap
+debops $@ bootstrap || debops --user root $@ bootstrap
```

Comment:
In the most common use case, where the debops/ansible user is unequal `root`, specially while provisioning a fresh VM/Server, we need to try to connect as the `ansible_user` and if it fails `||` (OR) will catch it and try to connect as user `root`. Therefore we still need this logic here.